frontend/src/views/Quiz.vue

```diff
@@ -23,12 +25,27 @@ onBeforeMount(() => {
         .then((response) => {
             if (response.error.value) {
                 errorOccurred.value = true;
-                console.log(response.error.value);
+                log(response.error.value);
             } else {
                 store.commit(
                     MutationTypes.UPDATE_CURRENT_QUIZ,
                     response.data.value
                 );
+
```

Comment:
hummm. this looks like a lot. the goal was to hide everything in the `useFetch` hook. We should really keep it simple in the component.

Like:
```typescript
const { data, error } = useFetch<T>(url, options);

// Use data somewhere
```